packages/apify/src/utils.ts

```diff
@@ -80,13 +80,8 @@ const psTreePromised = util.promisify(psTree);
  * NPM package, and it is automatically configured using the `APIFY_API_BASE_URL`, and `APIFY_TOKEN`
  * environment variables. You can override the token via the available options. That's useful
  * if you want to use the client as a different Apify user than the SDK internals are using.
- *
- * @param {object} [options]
- * @param {string} [options.token]
- * @param {string} [options.maxRetries]
- * @param {string} [options.minDelayBetweenRetriesMillis]
  */
-export function newClient(options = {}): ApifyClient {
+export function newClient(options: Omit<ApifyClientOptions, 'baseUrl' | 'requestInterceptors' | 'timeoutSecs'> = {}): ApifyClient {
```

Comment:
maybe we should use the whole `ApifyClientOptions`? `baseUrl` is definitely supported, and given what this method is used for, instead of omitting props from there, i would loosen the `ow` validation to support everything from that interface

Reply:
The point here is again - what we expose to the docs and what we do not. Previously it was only certain props, but yeah, originally I used the whole `ApifyClientOptions`. Then I just thought I would keep it the same. Kinda the same with some conflicts that I resolved yesterday - I could just use the whole `ApifyClientOptions` interface there (now I opted for `CreatClientOptions` added by Vlad).

Reply:
Yeah I know, but I dont see any reason why we should not expose everything here, `newClient` is basically just a shortcut for `new ApifyClient`, so why not allow passing all the options that the class accepts. There is already a validation in the constructor, so I would be ok with dropping it from here too.

Reply:
aha, cool, got it